diff --git a/node_modules/react-native-reanimated/src/reanimated2/NativeReanimated/NativeReanimated.ts b/node_modules/react-native-reanimated/src/reanimated2/NativeReanimated/NativeReanimated.ts
index 326b2f8..aedff2b 100644
--- a/node_modules/react-native-reanimated/src/reanimated2/NativeReanimated/NativeReanimated.ts
+++ b/node_modules/react-native-reanimated/src/reanimated2/NativeReanimated/NativeReanimated.ts
@@ -11,7 +11,7 @@ export class NativeReanimated {
   private InnerNativeModule: any;
 
   constructor(native = true) {
-    if (global.__reanimatedModuleProxy === undefined) {
+    if (global.__reanimatedModuleProxy === undefined && native) {
       const { ReanimatedModule } = NativeModules;
       ReanimatedModule?.installTurboModule();
     }
